boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuppr.dts

```diff
@@ -56,3 +100,10 @@
 	pinctrl-1 = <&uart136_sleep>;
 	pinctrl-names = "default", "sleep";
 };
+
+&pwm130 {
```

Comment:
What about switching to pwm131? Thus, both cores could have it's own PWM.

pwm130 is the default instance on cpuapp
https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp.dts#L296

Reply:
Unfortunatelly, PWM131 does not have access to P9 where leds are, thus cannot be used in led samples

Reply:
Right, I forgot about that constraint.